src/util/util.ts

```diff
@@ -159,6 +159,12 @@ async function generatorRunAsync(iterator: Generator<any>): Promise<any> {
   }
 }
 
+function policySortByPriority(priorityIndex: number, tokens: Array<string>, policy: Array<Array<string>>): void {
```

Comment:
`tokens` is not in use?

Reply:
my fault, I refactor this function but forget to remove useless argument.